schematic/configuration.py

```diff
@@ -0,0 +1,98 @@
+import os
+import yaml
+
+
+class Configuration(object):
```

Comment:
I decided to implement the configuration as a class for a few reasons:
- Apparently, once you import a constants in Python, the value can't be changed later by the module. For instance, if `schematic.TOKEN_PICKLE` was set to `None` when `schematic` is imported but is later updated to the actual value when the user runs `load_config()`, `schematic.TOKEN_PICKLE` will still be set to `None`. There might be a workaround that would involve putting some import statements later in the Python file, but that's unconventional. 
- You can use the special object methods (such as `__getattribute__()` and `__getitem__()`) to do some cool tricks. See comments below for details. 
- When imported in another module or script, everything is encapsulated in one object. You don't have to import each config value separately or load the YAML file over and over again.